api/suite_test.go

```diff
@@ -97,6 +97,8 @@ func (s *S) SetUpSuite(c *check.C) {
 	app.AuthScheme = nativeScheme
 	p := app.Platform{Name: "zend"}
 	s.conn.Platforms().Insert(p)
```

Comment:
add this to test filtering of platforms in api_test. If there's a better way to do this let me know.